deployments/charts/kuma/README.md

```diff
@@ -11,20 +11,29 @@ A Helm chart for the Kuma Control Plane
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | global.image.registry | string | `"kong-docker-kuma-docker.bintray.io"` | Default registry for all Kuma Images |
+| global.image.tag | string | `nil` | The default tag for all Kuma images, which itself defaults to .Chart.AppVersion |
 | patchSystemNamespace | bool | `true` | Whether or not to patch the target namespace with the system label |
 | controlPlane.logLevel | string | `"info"` | Kuma CP log level: one of off,info,debug |
 | controlPlane.mode | string | `"standalone"` | Kuma CP modes: one of standalone,remote,global |
-| controlPlane.kdsGlobalAddress | string | `""` |  |
-| controlPlane.replicas | int | `1` | Number of replicas of the Kuma CP |
+| controlPlane.zone | string | `nil` | Kuma CP zone, if running multizone |
+| controlPlane.kdsGlobalAddress | string | `""` | Only used in `remote` mode |
+| controlPlane.replicas | int | `1` | Number of replicas of the Kuma CP. Ignored when autoscaling is enabled |
+| controlPlane.autoscaling.enabled | bool | `false` | Whether to enable Horizontal Pod Autoscaling, which requires the [Metrics Server](https://github.com/kubernetes-sigs/metrics-server) in the cluster |
+| controlPlane.autoscaling.minReplicas | int | `2` | The minimum CP pods to allow |
+| controlPlane.autoscaling.maxReplicas | int | `5` | The max CP pods to scale to |
+| controlPlane.autoscaling.targetCPUUtilizationPercentage | int | `80` | For clusters that don't support autoscaling/v2beta, autoscaling/v1 is used |
+| controlPlane.autoscaling.metrics | list | `[{"resource":{"name":"cpu","target":{"averageUtilization":80,"type":"Utilization"}},"type":"Resource"}]` | For clusters that do support autoscaling/v2beta, use metrics |
```

Comment:
OK, how are we setting these on the command line? is it even possible?

Reply:
This is set dynamically by helm in the [`Capabilities.APIVersions` builtin object](https://helm.sh/docs/chart_template_guide/builtin_objects/) based on what is available in the target cluster. I think it is similar to the `kubectl api-versions` command.